From e3baaa14d1fedf889e31246cb7bdcc925589cc9a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 17 Oct 2021 12:06:06 +0200
Subject: [PATCH] knot-resolver & lighttpd: change meson location

Due to upstream change, we need to move this.

Fixes:
Makefile:29: /turris-os-packages-crashlab-mox/build/feeds/packages/devel/meson/meson.mk: No such file or directory
make[1]: *** No rule to make target '/turris-os-packages-crashlab-mox/build/feeds/packages/devel/meson/meson.mk'.  Stop.
---
 net/knot-resolver/Makefile | 2 +-
 net/lighttpd/Makefile      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/knot-resolver/Makefile b/net/knot-resolver/Makefile
index bfdddb7e2..b88f37dbf 100644
--- a/net/knot-resolver/Makefile
+++ b/net/knot-resolver/Makefile
@@ -26,7 +26,7 @@ PKG_BUILD_DEPENDS:=meson/host
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
-include $(TOPDIR)/feeds/packages/devel/meson/meson.mk
+include $(INCLUDE_DIR)/meson.mk
 
 define Package/knot-resolver
   SECTION:=net
diff --git a/net/lighttpd/Makefile b/net/lighttpd/Makefile
index 1a7958530..394d30611 100644
--- a/net/lighttpd/Makefile
+++ b/net/lighttpd/Makefile
@@ -28,7 +28,7 @@ PKG_CONFIG_DEPENDS:=CONFIG_LIGHTTPD_SSL $(patsubst %,CONFIG_PACKAGE_lighttpd-mod
 REBUILD_MODULES=authn_gssapi authn_ldap magnet webdav
 
 include $(INCLUDE_DIR)/package.mk
-include $(TOPDIR)/feeds/packages/devel/meson/meson.mk
+include $(INCLUDE_DIR)/meson.mk
 
 define Package/lighttpd/Default
   SUBMENU:=Web Servers/Proxies
-- 
2.30.2

